das-dashboard/backend/main.py

```
@@ -20,6 +23,11 @@ async def lifespan(app: FastAPI):
     WORKSPACE_SERVICES.ensure_workspace()
     WEB_CONFIG.load_user_profile()
     WEB_CONFIG.load_config_dictionary()
+    if os.path.exists(CONFIG_PATH):
+        try:
+            set_das_cli_config(CONFIG_PATH, web_config=WEB_CONFIG)
+        except Exception:
```

Pylint (4.0.6)

[warning] Catching too general exception Exception

(W0718)